kafka-connect-s3/pom.xml

```diff
@@ -61,6 +61,14 @@
         <dependency>
             <groupId>com.amazonaws</groupId>
             <artifactId>aws-java-sdk-s3</artifactId>
+            <exclusions>
+            <exclusion>
+                <!-- This is not used by the connector and is removed to address CVE-2020-28491
+                    Once the jackson version gets upgraded consider lifting this exclusion -->
```

Comment:
Just curious, why would we need to remove the exclusion after the Jackson version is upgraded if it's not currently used anyway?

Reply:
We don't necessarily want maintain this short of bookkeeping, with explicit exclusions that are too specific. But, yeah, we'll see.